docs/volunteering/tasks.md

```diff
@@ -0,0 +1,43 @@
+# Tasks
+
+We record tasks which need doing at https://github.com/srobo/tasks/issues.
+
+If you spot a task which you'd like to do, please `@mention` the person who
+created the task to let them know that you're interested. If you're able to
+self-assign the task (for technical reasons you won't initially be able to do
+this), please also do that.
+
+<!--- Review question: is the original author the right person to ping here? --->
```

Comment:
(obviously this line will be removed before merging)

Thoughts on this?

My reasoning is that the author is likely able to pass the volunteer on to someone else.
I am however a little concerned that this might put people off opening issues if they feel they'll end up needing to own the issue somewhat.

Reply:
It's not ideal, but i doubt there's an alternative. Having the person open them tag the relevant coordinator might work, but some things will transcend. 

Would relying on the fact that those who will likely be tagged are also likely watching the repo be enough? I somewhat doubt so? Alternatively someone watching can point in the right direction? eg:

> Person 1: What needs to be done to do this?
Person 2: @Person_3 will know how to do that
Person 3: Yes, do ...

It feels a little bureaucratic, but would probably work well. Obviously if 'Person 2' can answer the question, they should.

Reply:
38c4d486c6b4af86e14368935e76e20afb8e6bb2 expands on this a bit. I'm not completely happy with it, but I'm not sure of a better suggestion to make or a better way to frame it.

Suggestions for improvements are welcome, though I suggest that unless there's anything major we go with this for a while and if a better solution appears fix it then.